apps/web/app/(app)/environments/[environmentId]/(actionsAndAttributes)/actions/components/ActionClassesTable.tsx

```diff
@@ -45,16 +46,21 @@ export default function ActionClassesTable({
 
   return (
     <>
-      <LoadingWrapper isLoading={isLoading} error={error}>
+      <LoadingWrapper isLoading={false} error={error}>
```

Comment:
thanks for your contribution! feel wrong to hard code it to false 🤓

Reply:
Totally Agreed!
Thanks for noticing @jobenjada!
I have removed it in the next commit.